app/controllers/api/v2/roles_controller.rb

```diff
@@ -39,7 +41,10 @@ def create
       param_group :role
 
       def update
-        process_response @role.update_attributes(role_params)
+        taxonomies = taxonomy_params!
+        result = @role.update_attributes(role_params)
+        @role.set_taxonomies(taxonomies)
```

Comment:
I'm asking if you're not filtering (as in, `role_params_filter` should filter these attributes with `add_taxonomix_params_filter(filter)`) purposefully in order to handle taxonomies through `set_taxonomies`. 

It looks like putting business logic in the controller that the model should handle